software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -54,7 +54,8 @@ public void install() {
             executeCommandInTask(
                     getEntity().getConfig(VanillaWindowsProcess.INSTALL_COMMAND),
                     getEntity().getConfig(VanillaWindowsProcess.INSTALL_POWERSHELL_COMMAND),
-                    "install-command");
+                    "install-command",
+                    getEntity().getConfig(VanillaWindowsProcess.SUPPRESS_INSTALL_FAILURE));
```

Comment:
The `getConfig` call can return null. We should do `Boolean.TRUE.equals(getEntity().getConfig(VanillaWindowsProcess.SUPPRESS_INSTALL_FAILURE))` to avoid the risk of a `NullPointerException`.